docs/index.d.ts

```diff
@@ -1816,7 +1828,7 @@ export interface RosettaSignature {
  */
 export interface SigningPayload {
   /**
-   * The network-specific address of the account that should sign the payload.
+   * [DEPRECATED by account_identifier in v1.4.4] The network-specific address of the account that should sign the payload.
```

Comment:
This seems like a rebase problem?

Reply:
Yes, my branch was rebased with the master. I did some schema updates and ran `npm run build`. These changes came from that. 
If I run `npm run build` in the master branch, it is also giving me these changes.

Reply:
Ah see, thanks for the clarification.